src/passes/referencedLibraries.ts

```diff
@@ -0,0 +1,34 @@
+import assert = require('assert');
+import { ContractDefinition, ContractKind, FunctionCall, MemberAccess } from 'solc-typed-ast';
+import { AST } from '../ast/ast';
+import { ASTMapper } from '../ast/mapper';
+
+export class ReferencedLibraries extends ASTMapper {
+  visitFunctionCall(node: FunctionCall, ast: AST): void {
+    if (node.vExpression instanceof MemberAccess) {
+      const contractDef = new Map();
```

Comment:
ContractDef should have a type annotation to specify the key and value types of the map. This would improve readability and remove the need for asserting the type of the node later on